src/content/en/2021/cdn.md

```diff
@@ -281,14 +288,16 @@ This is quite important when you look at the fact that all sites will have to be
 
 ## HTTP/2+ (HTTP/2 or better) adoption
 
-HTTP/2 was introduced with a lot of hype and expectation. This was because the application layer protocol has not been updated since HTTP 1.1 in 1997. Since then, the web traffic trend, content-type, content size, website design, platforms, mobile apps etc., have evolved significantly. Thus, there was a need to have a protocol which can meet the requirements of the modern day web traffic and that protocol was going to be HTTP/2.
+HTTP/2 was introduced with a lot of hype and expectation. This was because the application layer protocol has not been updated since HTTP 1.1 in 1997. Since then, the web traffic trend, content-type, content size, website design, platforms, mobile apps and more have evolved significantly. Thus, there was a need to have a protocol which can meet the requirements of the modern day web traffic and that protocol was going to be HTTP/2.
 
 However, the implementation challenges of HTTP/2 features discouraged adoption. In addition, the net performance gains which can be expected with these changes was also not clear.
 
 This was where the CDNs became an intermediary in bridging the challenge of HTTP/2 implementation for web owners. Similar to TLS, an HTTP/2 connection terminates at the CDN level and this provides web owners the ability to deliver their website and subdomains over HTTP/2 without the need to upgrade their infrastructure to support it.
 
 CDNs acted as the proxy to bridge the gap by providing a layer to consolidate hostnames and route traffic to relevant endpoints with minimal change to their hosting infrastructure. Features like prioritizing content in the queue, server push could be managed from the CDN's side and a few CDN's even provided hands-off automated solutions to run these features without any inputs from website owners, thus providing a push to HTTP/2 adoption (no pun intended ;)).
 
+{# TODO: are emojis against our editors guide? leaving a note here to follow up #}
```

Comment:
@tunetheweb thoughts on the line above where it says (no pun intended ;))?

Reply:
Not sure about the policy for this. I'm other either way

Reply:
I say remove if you don't mind.